.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 sudo: false
 before_install: npm install -g grunt-cli
 node_js:
-  - "4"
+  - "4.2.6"
```

Comment:
lets follow @JeffJacobson's suggestion and just use `"node"` to continually target the version in long term support.